### ISSUE TITLE
GNOME 41 compatibility

### ISSUE THREAD

**Issue by muhmah-motahari** [alexduf/gnome-github-notifications#37]
I tested your extension by changing the ```metadata.json``` file for GNOME 41 and it worked fine

```
"shell-version": ["3.20", "3.22", "3.24", "3.26", "3.28", "3.30", "3.32", "3.34", "3.36", "3.38", "40", "41"]
```

**Comment by alexduf** [comment]
Hi, thanks for that.

How did you test it, by installing it from github or from the store?
The master branch on this repo has already got the fixes by @machitgarha, but when releasing it on to the store I introduced a regression for older versions of gnome. I've since then removed that specific version to avoid breakage on these gnome version.

I'm short in personal time to look into this, but two tickets are to be done before we can put a gnome 3.40+ compatible version back on the store: #34 and #35

Here's some background: https://github.com/alexduf/gnome-github-notifications/pull/33#issuecomment-921086004
